helium wallet app initial documentation (#892)
helium/docs: docs/wallets/helium-wallet-app.mdx

---
id: helium-wallet-app
sidebar_label: Helium Wallet App
slug: /wallets/helium-wallet-app
---

import useBaseUrl from "@docusaurus/useBaseUrl";

# Helium Wallet App

<img
  className="docsheader"
  src={useBaseUrl("img/wallets/helium-wallet-app/header.png")}
/>

> While you can import wallets that contain Hotspots and Validators, the Helium Wallet

Reviewer: Feels abrupt to introduce the app with a "but you can't do this".

Suggest something like 
"The Helium Wallet app is the new trusted way to manage your tokens and identity on the Helium Blockchain. This new Wallet App is not meant to manage Hotspots or Validators."

App is not for Hotspot/Validator management. It is for interacting with the HNT
wallet itself.

All Hotspot Management (diagnostics, asserting location, adding Hotspots, etc) should
be done via the [Maker Apps](/mine-hnt/maker-apps) available from the manufacturer.

## Wallets and Accounts

### Importing An Existing Wallet

<img
  className="appscreenbanner"
  src={useBaseUrl("img/wallets/helium-wallet-app/import-account-banner.png")}
/>

> If you have an existing HNT wallet, either one created from the original Helium app

Reviewer: ```suggestion
If you have an existing Helium wallet, either one created from the original Helium app
```

or using CLI, you can import it into the Wallet app.

|  Account Type  | Action                                                                                |
| :------------: | :------------------------------------------------------------------------------------ |
| 12 word wallet | Tap the 12 Words button and enter in the 12 word seed phrase, in the original order.  |
| 24 word wallet | Tap the 24 Words button and enter in the 24 word seed phrase, in the original order.  |
|     Ledger     | Tap the Ledger option and follow the in-app prompts to pair with the hardware wallet. |

### Creating A New Wallet

<img
  className="appscreenbanner"
  src={useBaseUrl("img/wallets/helium-wallet-app/new-account-banner.png")}
/>

> Wallets created with the Helium Wallet app utilize a 24 word key. These 24 words in

Reviewer: ```suggestion
Wallets created with the Helium Wallet app are generated with a 24 word seed phrase. These 24 words in
```

correct order are the only way to access the wallet, and can never be recovered if

> lost. They should never be shared, as whoever has them is considered the owner of

Reviewer: ```suggestion
lost. **They should never be shared**, as whoever has them is considered the owner of
```

the HNT included. Anyone asking for them is guaranteed to be a scammer.

> Taking and storing screenshots of the seed words is not recommended, as it is not a

Reviewer: ```suggestion
**Taking and storing screenshots of the seed words is not recommended, as it is not a**
```

> secure way to store them.

Reviewer: ```suggestion
secure way to store them. Screenshots and photos stored or backed up in the cloud can easily be hacked and funds stolen.
```

  src={useBaseUrl("img/wallets/helium-wallet-app/pair-ledger-banner.png")}
/>

Import your Ledger account by pairing with it over Bluetooth. Payment and other
transaction support will be coming soon.

## Working With Multiple Wallets

You are able to create/import multiple wallets, each with their unique set of 12 or 24
words. You can move between wallets by swiping left/right on the wallet name at the top of
the screen. When you tap on Settings, they will be for the selected wallet.

:::info How Many Wallets?
There is a limit of 10 wallet accounts (testnet + mainnet combined) per install.
:::

## Send And Request Tokens

<img
  className="appscreenbanner"
  src={useBaseUrl("img/wallets/helium-wallet-app/send-request-banner.png")}
/>

The Blue Send button allows you to choose the imported wallet you wish to send from,
and choose a Contact you wish to send HNT to.

The Green Request button allows you to quickly generate a QR code for someone to scan
(which provides your wallet address), or generate a link to share via email or text.
You can also enter in the amount you are requesting, if it is a known quantity, and add a note.

### Multiple Recipient Payment Support

> Long available on the CLI wallet, multiple recipient payments are now supported on the

Reviewer: ```suggestion
Already available on the CLI wallet, multiple recipient payments are now supported on the
```

Wallet app. This is most useful for those that need to wait for a payment transaction
to clear and appear on the blockchain before sending the next payment.

:::tip

You can create your own [QR codes](/wallets/app-wallet/deep-links-and-qr-codes/) to
scan, which can include an address/alias to send payment to.

> Payments-v2 can have multiple payees which can also be included in a QR code.

Reviewer: ```suggestion
The new payment type supported in the Helium Wallet app can have multiple payees which can also be included in a QR code.
```

> If you form a transaction in a QR code you can scan it to send, which can also be done

Reviewer: this reads a bit weird. maybe like "You can also form a Burn transaction in QR Code format."
